enclave/core/sgx/snmalloc/snmalloc_wrapper.cpp

```diff
@@ -19,11 +19,72 @@ void* ThreadAllocUntyped::get()
 
 #define USE_RESERVE_MULTIPLE 1
 #define IS_ADDRESS_SPACE_CONSTRAINED
+
+// In standard configuration, snmalloc defines the oe_* symbols
+// directly. In debug configuration, snmalloc defines _oe_* symbols
+// instead, and oe_* symbols are directed to the debug shim instead.
+#if defined(OE_USE_DEBUG_MALLOC)
+#define SNMALLOC_NAME_MANGLE(a) _oe_##a
```

Comment:
Couldn't this line be:
```suggestion
#define SNMALLOC_NAME_MANGLE(a) oe_nodebug_##a
```

Reply:
Then you can remove the later definitions in this case.

Reply:
Absolutely, thank you!